targetplatform/smarthome.target

```diff
@@ -1,91 +1,91 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I hope this file only ended up in this PR by accident?

Reply:
Yes, eclipse insists on changing an imported target platform :-(